crates/re_space_view_bar_chart/src/space_view_class.rs

```diff
@@ -282,3 +281,12 @@ impl SpaceViewClass for BarChartSpaceView {
         Ok(())
     }
 }
+
+fn to_egui_lot_corner(value: LegendCorner) -> egui_plot::Corner {
```

Comment:
typo :sob:

Reply:
I would have fixed this quickly, but someone has force-pushed to my branch :P

Reply:
:ship: :ship: :ship: